src/views/tickets/email.php

```diff
@@ -314,6 +314,7 @@
 				<table class="content" align="center" width="620" cellspacing="0" cellpadding="0" border="0" bgcolor="#ffffff" style="margin:0 auto; padding:0;<?php echo $break; ?>">
 					<tr>
 						<td align="center" valign="top" class="wrapper" width="620">
+							<?php do_action( 'tribe_tickets_ticket_email_ticket_top', $ticket ); ?>
```

Comment:
We want to add a docblock to this action.
E.g.: https://github.com/moderntribe/event-tickets/blob/master/src/Tribe/Tickets_Handler.php#L1123-L1132

Reply:
@bordoni thanks G. Docblock added